### PR TITLE
Clean up docker chain of filter table as well on driver init

### PR DESCRIPTION
**Pull request by chenchun**
Signed-off-by: Chun Chen <ramichen@tencent.com>

Relates to https://github.com/docker/libnetwork/pull/301.
I've seen some dirty rules left on filter table as well. I think it's better to clean up those rules.

### CODE REVIEW

**Comment by aboch** [comment]
@chenchun Can you please rebase

**Comment by chenchun** [comment]
@aboch Rebased.

File: drivers/bridge/bridge.go [hunk 1]
```diff
@@ -135,7 +135,10 @@ func Init(dc driverapi.DriverCallback, config map[string]interface{}) error {
 		logrus.Debugf("Fail to initialize firewalld: %v, using raw iptables instead", err)
 	}
 	if err := iptables.RemoveExistingChain(DockerChain, iptables.Nat); err != nil {
-		logrus.Warnf("Failed to remove existing iptables entries in %s : %v", DockerChain, err)
+		logrus.Warnf("Failed to remove existing iptables entries in table %s chain %s : %v", iptables.Nat, DockerChain, err)
+	}
+	if err := iptables.RemoveExistingChain(DockerChain, iptables.Filter); err != nil {
+		logrus.Warnf("Failed to remove existing iptables entries in table %s chain %s : %v", iptables.Filter, DockerChain, err)
 	}
```
Comment (aboch) [inline drivers/bridge/bridge.go]: Can you also please remove the isolation chain.
We can probably group the three cleanup calls in a function in `setup_ip_tables.go`

Comment (chenchun) [inline drivers/bridge/bridge.go]: updated.

File: drivers/bridge/setup_ip_tables.go [hunk 2]
```diff
@@ -15,6 +15,12 @@ const (
 	IsolationChain = "DOCKER-ISOLATION"
 )
 
+var bridgeChain = []iptables.ChainInfo{
```
Comment (aboch) [inline drivers/bridge/setup_ip_tables.go]: Please define these inside `removeIPChains()`, I don't think we need them as global vars.

**Comment by aboch** [comment]
Thanks.
Small nit, otherwise LGTM.

**Comment by aboch** [comment]
LGTM

**Comment by mrjana** [comment]
LGTM

**Pull request merged**